v5/tutorials/topical/adi.rst

```diff
@@ -73,10 +73,10 @@ line trackers.
          :caption: initialize.c
          :linenos:
 
-         #define ANALOG_SENSOR_PORT 1
+         #define ANALOG_SENSOR_PORT 'A'
 
          void initialize() {
-           adi_port_set_config(ANALOG_SENSOR_PORT, E_ADI_ANALOG_IN);
+           adi_port_set_config(ANALOG_SENSOR_PORT, ADI_ANALOG_IN);

```

Comment:
was the previous code wrong? afaik this only works if the user has `USE_SIMPLE_NAMES` or whatever the symbol was defined.

unless there's some urgent reason to change this I think I'd rather it stayed as it was